src/api-reference/user-provisioning/v4.spend-user-provisioning.md

```diff
@@ -0,0 +1,702 @@
+                    ]
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:WorkflowPreference": {
+                    "emailStatusChangeOnCashAdvance": true,
+                    "emailAwaitApprovalOnCashAdvance": true,
+                    "emailStatusChangeOnReport": true,
+                    "emailAwaitApprovalOnReport": true,
+                    "promptForApproverOnReportSubmit": true,
+                    "emailStatusChangeOnTravelRequest": true,
+                    "emailAwaitApprovalOnTravelRequest": true,
+                    "promptForApproverOnTravelRequestSubmit": true,
+                    "emailStatusChangeOnPayment": true,
+                    "emailAwaitApprovalOnPayment": true,
+                    "promptForApproverOnPaymentSubmit": true
+                },
+                "urn:ietf:params:scim:schemas:extension:spend:2.0:UserPreference": {
+                    "allowCreditCardTransArrivalEmails": true,
+                    "allowReceiptImageAvailEmails": true,
+                    "promptForCardTransactionsOnReport": true,
+                    "autoAddTripCardTransOnReport": true,
+                    "promptForReportPrintFormat": true,
+                    "defaultReportPrintFormat": "DETAILED",
+                    "showTotalOnReport": true,
+                    "showExpenseOnReport": "ALL",
+                    "showInstructHelpPanel": true,
+                    "showImagingIntro": true,
+                    "expenseAuditRequired": "REQUIRED",
+                    "useQuickItinAsDefault": true
+                },
+                "urn:ietf:params:scim:schemas:extension:enterprise:2.0:Payroll": {
+                    "adp": {
+                        "companyCode": "abcd",
+                        "deductionCode": "abcd",
+                        "employeeFileNumber": "1234"
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+## <a name="create-bulk-provision-patch-request"></a>`PATCH provisioning/v4/Bulk` endpoint for managing spend data
+
+Updates one or more provisioning request using the `provision/v4/Bulk` endpoint.
+PATCH requests modify the resource within Concur.
+
+### Scopes
+
+- `spend.user.general.writeonly`
+
+Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+```shell
+POST https://www.us.api.concursolutions.com/provisioning/v4/Bulk/
+```
+
+#### URI
+
+```shell
+POST /provisioning/v4/Bulk/
+```
+
+#### Headers
+
+- `concur-correlationid` is a SAP Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+
+#### PATCH Payload Example
+
+- Note that Concur UUID must be in the path of the request.
+
+```shell
+{
+    "schemas": [
+        "urn:ietf:params:scim:api:messages:2.0:BulkRequest",
+        "urn:ietf:params:scim:api:messages:2.0:PatchOp"
+    ],
+    "failOnErrors": 1,
+    "Operations": [
```

Comment:
Can we provide a more complex example too (one where the path has `[` or something like for Approvers? Can show that we accept more than a single Operation perhaps/